docs_user/assemblies/assembly_migrating-ceph-rbd.adoc

```diff
@@ -4,13 +4,40 @@
 
 = Migrating Red Hat Ceph Storage RBD to external RHEL nodes
 
-For hyperconverged infrastructure (HCI) or dedicated Storage nodes that are running {Ceph} version 6 or later, you must migrate the daemons that are included in the {rhos_prev_long} control plane into the existing external Red Hat Enterprise Linux (RHEL) nodes. The external RHEL nodes typically include the Compute nodes for an HCI environment or dedicated storage nodes.
+For hyperconverged infrastructure (HCI) or dedicated Storage nodes that are
+running {Ceph} version 6 or later, you must migrate the daemons that are
+included in the {rhos_prev_long} control plane into the existing external Red
+Hat Enterprise Linux (RHEL) nodes. The external RHEL nodes typically include
+the Compute nodes for an HCI environment or dedicated storage nodes.
 
-To migrate Red Hat Ceph Storage Rados Block Device (RBD), your environment must meet the following requirements:
+To migrate Red Hat Ceph Storage Rados Block Device (RBD), your environment must
+meet the following requirements:
 
 * {Ceph} is running version 6 or later and is managed by cephadm/orchestrator.
-* NFS (ganesha) is migrated from a {OpenStackPreviousInstaller}-based deployment to cephadm. For more information, see xref:creating-a-ceph-nfs-cluster_migrating-databases[Creating a NFS Ganesha cluster].
-* Both the {Ceph} public and cluster networks are propagated, with {OpenStackPreviousInstaller}, to the target nodes.
-* Ceph Monitors need to keep their IPs to avoid cold migration.
+* NFS (ganesha) is migrated from a {OpenStackPreviousInstaller}-based
+  deployment to cephadm. For more information, see
+  xref:creating-a-ceph-nfs-cluster_migrating-databases[Creating a NFS Ganesha
+  cluster].
+* Both the {Ceph} public and cluster networks are propagated, with
+  {OpenStackPreviousInstaller}, to the target nodes.
+* Ceph MDS, Ceph Monitoring stack, Ceph MDS, Ceph RGW and other services have
+  been migrated already to the target nodes;
+ifeval::["{build}" != "upstream"]
+* The daemons distribution follows the cardinality constraints described in the
+  doc link:https://access.redhat.com/articles/1548993[Red Hat Ceph Storage:
+  Supported configurations]
+endif::[]
+* The Ceph cluster is healthy, and the `ceph -s` command returns `HEALTH_OK`
 
-include::../modules/proc_migrating-mon-and-mgr-from-controller-nodes.adoc[leveloffset=+1]
+The high level procedure that migrates the Ceph Mon daemons is based on th
+following assumptions:
+
+* It keeps the mon IP addresses by moving them to the target {Ceph} nodes
+* It drains the existing Controller nodes that are supposed to be decommisioned
+* It deploys additional monitors to the target nodes, and promote them as
+`_admin` nodes that administrators can use to manage the {CephCluster} cluster
+and perform day 2 operations.
```

Comment:
I'm a little confused about the intent of this list. 

- Does "it" refer to the procedure?
- The "assumptions" look like a list of actions that occur during the procedure. Maybe something like, "During the high-level procedure to migrate the Ceph Mon daemons, the following actions occur:"

Reply:
ack and thanks for the input! I'll rewrite this block to make it clear what's the intent, and yes, those are high-level actions that describe the procedure that will be described later with more details.